techniques/css/C42.html

```diff
@@ -18,67 +18,87 @@ <h2>When to Use</h2>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this technique is to ensure that links in navigation or pagination menus will be spaced so that they fall within an area that measures at least 44 x 44 CSS pixels if the target area itself is smaller than that. The aim is to provide an adequate target clearance so the offset to adjacent targets is sufficient to prevent accidental pointer activation of adjacent targets.</p>	
-		
+			<p>The objective of this technique is to ensure that navigation or pagination links will be spaced so that they fall within an area that measures at least 44 &#215; 44 CSS pixels if the target area itself is smaller than that. The aim is to provide an adequate target clearance so the offset to adjacent targets is sufficient to prevent accidental pointer activation of adjacent targets.</p>
```

Comment:
I think the "if the target area is smaller than that" bit was meant for the other technique - it doesn't belong in this one, and it needs to be in the other one but is missing there.

Reply:
Fixed. I took out "if the target area itself is smaller than that" as that feels redundant